src/registrar/config/settings.py

```diff
@@ -356,9 +356,18 @@
 # strict CSP by allowing scripts to run from their domain
 # and inline with a nonce, as well as allowing connections back to their domain.
 # Note: If needed, we can embed chart.js instead of using the CDN
-CSP_SCRIPT_SRC_ELEM = ["'self'", "https://www.googletagmanager.com/", "https://cdn.jsdelivr.net/npm/chart.js"]
-CSP_CONNECT_SRC = ["'self'", "https://www.google-analytics.com/"]
-CSP_INCLUDE_NONCE_IN = ["script-src-elem"]
+CSP_DEFAULT_SRC = ("'self'",)
+CSP_STYLE_SRC = ["'self'", "https://www.ssa.gov"]
+CSP_SCRIPT_SRC_ELEM = [
+    "'self'",
+    "https://www.googletagmanager.com/",
+    "https://cdn.jsdelivr.net/npm/chart.js",
+    "https://www.ssa.gov",
+    "https://ajax.googleapis.com",
```

Comment:
If we remove `"https://ajax.googleapis.com"`, we get the error of

```
 andi.js:4143 Refused to load the script 'https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js' because it violates the following Content Security Policy directive: "script-src-elem 'self' https://www.googletagmanager.com/ https://cdn.jsdelivr.net/npm/chart.js https://www.ssa.gov".
 ```

(I think they need the googleapi lib to render and/or talk to ANDI?)